### PR TITLE
Tenant webhook doc updates

### PR DESCRIPTION
**Pull request by robfusion**
--https://github.com/FusionAuth/fusionauth-issues/issues/1812

### CODE REVIEW

File: site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc [hunk 1]
```diff
@@ -18,7 +18,8 @@ Additional headers may be added to the request by adding headers to the Webhook
 * <<Configuration>>
 ** <<Application Scoped Events>>
 ** <<Tenant Scoped Events>>
-** <<Example Configuration>>
+** <<Example Configuration Before 1.37.0>>
```
Comment (mooreds) [inline site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc:21]: I would move the pre 1.37 configuration below the post 1.37 configuration.

**Review by mooreds — CHANGES_REQUESTED** [review]
* url bar is a bit weird on the site/assets/img/docs/core-concepts/tenant-configuration-webhooks-settings.png file. Can you retake the screenshot without showing the full url?
* all the webhook settings images should be retaken without the `applications` tab (except of course the applications tab image)

File: site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc [hunk 2]
```diff
@@ -36,7 +37,9 @@ Then, configure the Tenant to listen for the event by navigating to [breadcrumb]
 
 If you have multiple Webhooks configured for a single Tenant, the transaction setting for the event will dictate if FusionAuth will commit the transaction or not.
 
-If you have multiple tenants listening for the same event, they will all receive that event and can filter on the provided [field]#tenantId# to determine if they should handle the event.
+As of version 1.37.0 if you have multiple tenants listening for the same event, they will only receive the events for that tenant, but not for other tenants.
```
[the remark this answers is not in the document]
Comment (robfusion) [reply site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc:40]: Basically, if two tenants both have user.create events enabled, but are using different webhooks. Tenant A's webhook will not get called for tenant B user.create events, and vice versa. Agree the sentence is confusing.

File: site/docs/v1/tech/events-webhooks/index.adoc [hunk 3]
```diff
@@ -30,6 +30,7 @@ Continue reading below to see how the events and webhooks are configured using t
 
 * <<Tenant Settings>>
 * <<Add Webhook>>
+* <<Tenants>>
```
[the remark this answers is not in the document]
Comment (robfusion) [reply site/docs/v1/tech/events-webhooks/index.adoc:33]: I added that so I could ink directly here from tenants.adoc Webhooks section. I just thought that was more convenient.

**Comment by robfusion** [comment]
> 

Fixed tenant config to have the pretty URL.
Updated webhook setting images to show new tenants tab.

File: site/docs/v1/tech/events-webhooks/index.adoc [hunk 4]
```diff
@@ -154,6 +156,29 @@ Not all events are considered application specific and selecting an application
 +
 In most cases you will want to use the [field]#All applications# configuration.
 
+=== Tenants
+Available since 1.37.0
```
[the remark this answers is not in the document]
Comment (robfusion) [reply site/docs/v1/tech/events-webhooks/index.adoc:137]: Done

File: site/docs/v1/tech/events-webhooks/index.adoc [hunk 5]
```diff
@@ -131,6 +132,7 @@ The SSL certificate in PEM format to be used when connecting to the webhook. Whe
 
 
 === Applications
```
[the remark this answers is not in the document]
Comment (robfusion) [reply site/docs/v1/tech/events-webhooks/index.adoc]: Done

File: site/docs/v1/tech/events-webhooks/index.adoc [hunk 6]
```diff
@@ -154,6 +156,29 @@ Not all events are considered application specific and selecting an application
...
++
+Most events are considered tenant specific and selecting a tenant will limit you to only receiving events for those tenants. The exceptions are the following instance events which are not tenant specific:
++
+- `audit-log.create`
+- `event-log.create`
+- `kickstart.success`
+
 === Headers
 
 image::webhook-settings-headers.png[Webhook Settings - HTTP Headers,width=1200,role=shadowed top-cropped]
```
[the remark this answers is not in the document]
Comment (robfusion) [reply site/docs/v1/tech/events-webhooks/index.adoc]: removed shadowed role

File: site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc (hunk 1 above)
Comment (robfusion) [reply site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc]: Done

File: site/docs/v1/tech/events-webhooks/index.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (mooreds) [reply site/docs/v1/tech/events-webhooks/index.adoc:33]: Ah, you can do this without adding the `<<Tenants>>` .

Any H2 or H3 should have an anchor tag added.

File: site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (mooreds) [reply site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc:40]: Maybe:

```
If you have multiple webhooks assigned to different tenants but configured for the same event, such as `user.create`, only the events matching both tenant and type will be delivered. For exampele, imagine you have Pied Piper and Hooli tenants and both have different webhooks (piedpier.com/webhook and hooli.com/webhook) and each is configured to listen only to their tenant for the `user.create` webhook event. In this case, piedpiper.com/webhook would receive only Pied Piper user creation event information; likewise hooli.com/webhook will receive only webhooks for the `user.create` event from the Hooli tenant. 
```

[the remark this answers is not in the document]
Comment (robfusion) [reply site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc]: Yeah, that's better.

File: site/docs/v1/tech/events-webhooks/index.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (robfusion) [reply site/docs/v1/tech/events-webhooks/index.adoc]: Ok, cool. Removed the TOC entry.

**Pull request merged**